packages/tupaia-web/src/layout/UserMenu/PopoverMenu.tsx

```diff
@@ -35,7 +35,11 @@ export const PopoverMenu = ({
 }: PopoverMenuProps) => {
   return (
     <Popover
-      PaperProps={{ style: { backgroundColor: primaryColor } }}
+      PaperProps={{
+        style: {
+          backgroundColor: '#2E2F33',
```

Comment:
Similarly, needs to be `primaryColor` but you can change the default primaryColor to be this black. The reason is that if we have a custom landing page that has, say, white set as the primary color, then that white won't get carried through if we set to be black all the time